### PR TITLE
Fix react warning

### LINKED ISSUE #2764: Fix React warnings in src/app/containers/Metadata/index.test.jsx
**Describe the bug**
There are React warnings in unit tests.

**To Reproduce**

1. Go to simorgh in terminal
2. Run npm run test:unit
3. Observe React warnings:

```
    console.log src/testHelpers/setupTests.js:19
      Test failed because too many React warnings were detected. Please fix the following:
      Warning: Failed prop type: Invalid prop `timeFirstPublished` of type `string` supplied to `Metadata`, expected `number`.
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: Invalid prop `timeFirstPublished` of type `string` supplied to `Metadata`, expected `number`.
    console.error src/testHelpers/setupTests.js:30
      Warning: Failed prop type: Invalid prop `timeLastPublished` of type `string` supplied to `Metadata`, expected `number`.
```

**Expected behaviour**

No React warning for test suite.

**Testing notes**

No testing required.

### PR DESCRIPTION
**Pull request by FK78**
Resolves #2764 

**Overall change:** _Change prop required data type to string_

**Code changes:**

- _Change timeFirstPublished data type_
- _Change timeLastPublished data type_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by thekp — APPROVED** [review]

File: src/app/components/Metadata/index.jsx [hunk 1]
```diff
@@ -172,9 +172,9 @@ Metadata.propTypes = {
     propTypeCheck(props, propName, 'Metadata', arrayOf(string)),
   themeColor: string.isRequired,
   timeFirstPublished: (props, propName) =>
-    propTypeCheck(props, propName, 'Metadata', number.isRequired),
+    propTypeCheck(props, propName, 'Metadata', string.isRequired),
```
Comment (jroebu14) [inline src/app/components/Metadata/index.jsx:175]: What exactly does this do? Can you find out if it adds any value? I don't think there should be abstractions over prop types. Prop types should be as explicit as possible.

**Review by jroebu14 — COMMENTED** [review]
Looks good 👍 I'm just suspicious about `propTypeCheck` it's possibly something we can remove to make prop types more explicit and not abstract away the component's API.

File: src/app/components/Metadata/index.jsx (hunk 1 above)
Comment (chris-hinds) [reply src/app/components/Metadata/index.jsx:175]: I agree, opening another issue to investigate why we are doing this would be a good idea.

**Review by chris-hinds — APPROVED** [review]

**Comment by chris-hinds** [comment]
@FK78 Can you add a comment on this PR https://github.com/bbc/simorgh/pull/2963 as it looks like you are both solving the same issue.

**Pull request merged**